testing/consul/APKBUILD

```diff
@@ -60,14 +36,18 @@ prepare() {
 }
 
 build() {
-	cd "$_builddir"/$pkgname || return 1
-	export GOPATH="$srcdir"
-	go build -v -o bin/consul \
-		-ldflags "-X main.GitDescribe $pkgver" || return 1
+	cd "$builddir"/$pkgname || return 1
+
+	mkdir -p "${builddir}/src/github.com/hashicorp"
+	ln -s "${builddir}" "${builddir}/src/github.com/hashicorp/consul"
+	cd "${builddir}/src/github.com/hashicorp/consul"
```

Comment:
This directory structure is inside the build folder (there is some nesting) and is how Go finds the packages/classes. Also i have another Go Hashicorp package (consul-template) coming up.